Add Apache instructions (#97369)
grafana/grafana: docs/sources/tutorials/run-grafana-behind-a-proxy/index.md

For more detailed instruction, refer to the [tutorial on IIS URL Rewrites](/tutorials/iis/).

### Configure Apache

> To use Apache as a proxy, first ensure that the Apache proxy module [mod_proxy](https://httpd.apache.org/docs/current/mod/mod_proxy.html) is installed and enabled. To enable, run the following commands:

Reviewer: ```suggestion
To use Apache as a proxy, ensure its proper installation and configuration.

1.  Ensure that the Apache proxy module [mod_proxy](https://httpd.apache.org/docs/current/mod/mod_proxy.html) is installed and enabled. To enable, run the following commands:
```

```bash
a2enmod proxy
a2enmod proxy_http
```

> To configure the proxy, edit the site configuration file. Inside the `<VirtualHost>` section, add the following:

Reviewer: ```suggestion
2. To configure the proxy, edit the site configuration file. To do so, inside the `<VirtualHost>` section, add the following code:
```

> ```

Reviewer: ```suggestion
```bash
```

  ProxyPreserveHost on
  ProxyPass / http://your_grafana_server:3000
  ProxyPassReverse / http://your_grafana_server:3000
```

> After this, you will need to restart Apache for the settings to take effect.

Reviewer: ```suggestion
3. Finally, you'll need to restart Apache for the settings to take effect.
```

> Once you've restarted, navigate to your Apache server on port 80 and you will be redirected to Grafana.

Reviewer: ```suggestion
Once you've restarted Grafana, navigate to your Apache server on port `80`. You're redirected to Grafana.
```

> To configure Grafana hosted in a sub path, you will need to replace this with the following (assuming your Grafana instance is on the sub path `your_path`):

Reviewer: ```suggestion
To configure Grafana hosted in a sub path, replace the sub path with the following code (assuming your Grafana instance is on the sub path `your_path`):
```

> ```

Reviewer: ```suggestion
```bash
```

For more detailed instruction, refer to the [tutorial on IIS URL Rewrites](/tutorials/iis/).

### Configure Apache

To use Apache as a proxy, ensure its proper installation and configuration.

> 1.  Ensure that the Apache proxy module [mod_proxy](https://httpd.apache.org/docs/current/mod/mod_proxy.html) is installed and enabled. To enable, run the following commands:

Reviewer 2: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#code

```suggestion
1.  Ensure that the Apache proxy module [`mod_proxy`](https://httpd.apache.org/docs/current/mod/mod_proxy.html) is installed and enabled. To enable, run the following commands:
```
